src/runtime/entitlements-manager.js

```diff
@@ -150,11 +152,13 @@ export class EntitlementsManager {
   /**
    * @param {string} raw
    * @param {boolean=} opt_isReadyToPay
+   * @param {?string} opt_decryptedDocumentKey
    * @return {boolean}
    */
-  pushNextEntitlements(raw, opt_isReadyToPay) {
+  pushNextEntitlements(raw, opt_isReadyToPay, opt_decryptedDocumentKey) {
```

Comment:
I don't think a decrypted document key could ever be available for the `push` operation. This comes from the payments flow which is completely document-agnostic.

Reply:
done